src/SFA.DAS.Admin.Aan.Web/Validators/ManageEvent/SchoolNameViewModelValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using SFA.DAS.Admin.Aan.Web.Models.ManageEvent;
+
+namespace SFA.DAS.Admin.Aan.Web.Validators.ManageEvent;
+
+public class SchoolNameViewModelValidator : AbstractValidator<SchoolNameViewModel>
+{
+    public const string EventSchoolNameEmpty = "You must include the name of the school";
+
+    public SchoolNameViewModelValidator()
+    {
+        RuleFor(e => e.SearchTerm)
+            .Must(SchoolNotEntered)
+                .WithMessage(EventSchoolNameEmpty);
+    }
+
+    private static bool SchoolNotEntered(SchoolNameViewModel model, string? searchTerm)
```

Comment:
Method name indicates opposite. Should be SchoolNameEntered. There is better way of doing this, you can access Urn from the expression in the `Must` by taking the model argument.